packages/warehouses/src/warehouseClients/DatabricksWarehouseClient.ts

```diff
@@ -396,6 +396,21 @@ export class DatabricksWarehouseClient extends WarehouseBaseClient<CreateDatabri
         }
     }
 
+    async getAllTables() {
+        const query = `
+            SELECT table_catalog, table_schema, table_name
+            FROM information_schema.tables
+            WHERE table_type = 'MANAGED' 
```

Comment:
I see, so we can include more tables depending on types here: https://docs.databricks.com/en/sql/language-manual/information-schema/tables.html#table-types


Will review this with the credentials you've set up 👍

Reply:
yeah, is there any other you think we should include ? like `materialized_view` I'd like to avoid `view` because that's what the system table uses

Reply:
Perhaps let's leave it for now? I like what you have already